docs/Users/Tutorials/Git_Help.rst

```diff
@@ -230,6 +220,27 @@ fine by our reviewers, you can simply send it again like this. If not, edit it
 and send it.
 Now you have successfully edited your last commit!
 
+Keeping your fork in sync
+-------------------------
+
+As people work on coala new commits will be added. This will result in your
+local fork going out of sync with the remote repository.
+To sync your changes with the remote repository run the following commands:
+
+::
+
+    $ git checkout master
+    $ git git pull origin master
+    $ git checkout branchname
+    $ git rebase master
+
+Explaination on what each command does can be found below.
+
+.. seealso::
+
+  For more information about syncing a fork, see
+  `Github's guide <https://help.github.com/articles/syncing-a-fork/>`_.
```

Comment:
Not needed. Your workflow is different, you might want to link to a couple of SO questions.